docs/specs/tripgo.swagger.yaml

```diff
@@ -3850,3 +3853,19 @@ definitions:
       exchange:
         description: Rate to convert cost from user's local currency to dollar (optional).
         type: number
+
+  EnabledMode:
+    description: MaaS bundle enabled mode for this segment
+    properties:
+      mode:
+        type: string
+        description: mode of transport
+      pointsPerCost:
+        type: number
+        description: 0 for unlimited/free, 1 by default, 0.9 for 10% discount
+      discountPoints:
+        type: number
+        description: in cents, 500 for 5 AUD fixed discount, not included if no discount
```

Comment:
I'm confused. Why is this in cents? Shouldn't it be in points? What if someone wants 1 point per dollar, or 10 points per dollar?

Reply:
agree we need better wording here, just tried to make it clear on how we handle it in the BE.